### ISSUE TITLE
Quick settings button

### ISSUE THREAD

**Issue by Commenter25** [Myzel394/Alibi#19]
A useful feature would be a quick settings button to save recording. This would likely be faster than opening the app, and could be done on the lock screen without unlocking the device. If holding the button opened the app, you could remove it from your home screen and app drawer, essentially "hiding" it in the quick settings.

**Comment by Myzel394** [comment]
Do you mean something like this when you long press on the app icon?

![signal-2023-08-29-212636](https://github.com/Myzel394/Alibi/assets/50424412/87cd53bb-b11c-40ac-9924-ddc3488797db)

**Comment by Commenter25** [comment]
No, I mean the quick settings above notifications, when you swipe down from the top of the screen.
